jupyter/datascience/ubi9-python-3.12/setup-elyra.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -x
+
+# Set the elyra config on the right path
+jupyter elyra --generate-config
+cp /opt/app-root/bin/utils/jupyter_elyra_config.py /opt/app-root/src/.jupyter/
+
+# create the elyra runtime directory if not present
+if [ ! -d $(jupyter --data-dir)/metadata/runtimes/ ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)